unitTests/HostMatrix.C

```diff
@@ -60,7 +61,7 @@ public:
         }
 
         word fieldName{"p"};
-        dimensionSet ds{};
+        dimensionSet ds{0,0,0,0,0};
```

Comment:
Why does `dimensionalSet` for pressure have zeros for all entries?

Reply:
because it is just a dummy value, but for the .org version of OF there is no default constructor. so we have to put in a value.